hw/dv/tools/dvsim/questa.hjson

```diff
@@ -2,26 +2,34 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  build_cmd:  "{job_prefix} {QUESTA_HOME}/questasim/linux_x86_64/qrun -optimize"
-  run_cmd:    "{job_prefix} {QUESTA_HOME}/questasim/linux_x86_64/qrun -simulate"
+  build_cmd:  "{job_prefix} {QUESTA_HOME}/linux_x86_64/qrun -optimize"
+  run_cmd:    "{job_prefix} {QUESTA_HOME}/linux_x86_64/qrun -simulate"
 
 
   build_opts: [ "-timescale 1ns/1ps",
                 "-outdir {build_dir}/qrun.out",
-                "-uvm -uvmhome {QUESTA_HOME}/questasim/verilog_src/uvm-1.2",
+				"+incdir+{QUESTA_HOME}/verilog_src/uvm-1.2/src",
+				"+incdir+{QUESTA_HOME}/verilog_src/uvm-1.2/src/mentor",
+                "{QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
+                "-uvm -uvmhome {QUESTA_HOME}/verilog_src/uvm-1.2",
                 "-mfcu",
+                "-sv",
+				"-svext=hierbinsof",
+                "-suppress 2744,2912,7070,7034,7033,2244,7041",
```

Comment:
I'm willing to believe this is needed, but could it have a comment above it that says roughly we're suppressing? Before this change, `run_opts` had a comment like that for the 8323 message that was being suppressed.

I think we will need equivalent notes for all the other warnings that are being suppressed.